README.md

```diff
@@ -4,9 +4,10 @@ An Ansible role for installing Docker.
 
 ## Role Variables
 
-- `docker_version` - Docker version
+- `docker_version` - Docker version (default `17.03.*`)
```

Comment:
I think that we'd want to default the version to something like `17.*`. Otherwise, it appears that we could be updating this version number every month.